script/dump-db

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+# must run as geekotest
+
+pg_dump -Fc -c openqa  -f ~/SQL-DUMPS/`date -Idate`.dump
+find ~/SQL-DUMPS/ -mtime +7 -print0 | xargs --no-run-if-empty -0 rm
```

Comment:
Having this in an actually maintained repo will only increase the chance that somebody accidentally copies this.

Maybe this makes it at least less dangerous:

```suggestion
find /var/lib/openqa/SQL-DUMPS -mtime +7 -print0 -delete
```

It would probably still make more sense to keep a certain number of dumps.

The line above should probably use the full path as well.